From ad1833c29ce2f61dc07c9837ccfb47ae7493a01c Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Tue, 31 Jul 2018 17:41:20 +0800
Subject: [PATCH 4366/5242] MLK-19111: ISI: fix module installation fail issue

commit  fc4775a9926b3fdcf291c6214190388b18aee807 from
https://source.codeaurora.org/external/imx/linux-imx.git

If build ISI driver as module and insmod it after system
boot up, it will fail. Because lack of MODULE_LICENSE("GPL")
,insmod isi driver will print many unknown symbol messages

Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
Reviewed-by: Robby.Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/imx8/mxc-isi-core.c |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/media/platform/imx8/mxc-isi-core.c b/drivers/media/platform/imx8/mxc-isi-core.c
index 7943802..dccb483 100644
--- a/drivers/media/platform/imx8/mxc-isi-core.c
+++ b/drivers/media/platform/imx8/mxc-isi-core.c
@@ -259,3 +259,9 @@ static int mxc_isi_runtime_resume(struct device *dev)
 };
 
 module_platform_driver(mxc_isi_driver);
+
+MODULE_AUTHOR("Freescale Semiconductor, Inc.");
+MODULE_DESCRIPTION("MXC Image Subsystem driver");
+MODULE_LICENSE("GPL");
+MODULE_ALIAS("ISI");
+MODULE_VERSION("1.0");
-- 
1.7.9.5

